include/zephyr/net/zperf.h

```diff
@@ -51,10 +51,10 @@ struct zperf_results {
 	uint32_t nb_packets_rcvd;
 	uint32_t nb_packets_lost;
 	uint32_t nb_packets_outorder;
-	uint32_t total_len;
-	uint32_t time_in_us;
+	uint64_t total_len;
+	uint64_t time_in_us;
 	uint32_t jitter_in_us;
-	uint32_t client_time_in_us;
+	uint64_t client_time_in_us;
```

Comment:
We are changing a struct that is used in an somewhat public API, so this change should be mentioned in the migration guide which could explain what is changed from user point of view.
The document is `doc/releases/migration-guide-3.7.rst`

Reply:
the doc is added, please review again